orderbook/src/database.rs

```diff
@@ -76,14 +131,129 @@ impl Database {
     }
 }
 
+#[async_trait::async_trait]
+impl Database for Postgres {
```

Comment:
~Why the extra level of indirection and not move the implementation directly here?~

I see now, because the methods are implemented in separate modules.